src/stactools/cli/commands/lint.py

```diff
@@ -0,0 +1,20 @@
+import click
+from stac_check.lint import Linter  # type: ignore
+
+
+def create_lint_command(cli: click.Group) -> click.Command:
+
+    @cli.command("lint", short_help="Lint a stac object with stac-check.")
+    @click.argument("href")
+    def lint_command(href: str) -> None:
+        """Lint a STAC object via stac-check.
+
+        Prints any warnings to stdout.
+        """
+        linter = Linter(href, assets=True, links=True, recursive=False)
+        for _, v in linter.create_best_practices_dict().items():
+            click.secho("WARNING", fg="yellow", nl=False)
+            for value in v:
+                click.echo(f" {value}")
```

Comment:
It'd be nice to have a green 🆗 if there are no lints found, that could be disabled with a `--quiet` flag.

Reply:
true